node/express/containerCosmosDBWithTests/Tests/package.json

```diff
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "applicationinsights": "^1.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
```

Comment:
Do we need to increase the patch version  of the package

Reply:
That will not be required as we do not need to publish this separately somewhere and have always maintained version 0 for it.